static/static_data.json

```diff
@@ -12,16 +12,16 @@
         "totalNewUndelivered": 238
     },
     "Green": {
-        "totalOldActive": 175,
-        "totalOldInactive": 22,
+        "totalOldActive": 178,
+        "totalOldInactive": 29,
         "totalNewDelivered": 24,
-        "totalNewUndelivered": 0
+        "totalNewUndelivered": 102
```

Comment:
Assume this is now for Type 10s? Originally these numbers were for the Type 9s

Reply:
Yep the roster is now listing Type 10s as undelivered. I know it's a long way off so we can ignore these numbers for now

Reply:
We should probably call this out? Tough with the Green Line since the 9's were the original new cars but now the 10s are on the horizon. Wdyt?